contribute/style-guides/frontend.md

```diff
@@ -7,27 +7,80 @@ Generally we follow the Airbnb [React Style Guide](https://github.com/airbnb/jav
 - [Frontend Style Guide](#frontend-style-guide)
   - [Table of Contents](#table-of-contents)
   - [Basic rules](#basic-rules)
-  - [Organization](#organization)
-  - [Props](#props)
+  - [Naming](#naming)
+  - [Files and directories naming](#files-and-directories-naming)
+  - [Code organization](#code-organization)
+    - [Exports](#exports)
+  - [Comments](#comments)
+  - [React](#react)
+    - [Props](#props)
   - [State management](#state-management)
 
 ## Basic rules
 
 - Try to keep files small and focused.
 - Break large components up into sub-components.
-
-## Organization
-
-- Components and types that needs to be used by external plugins needs to go into @grafana/ui
-- Components should get their own folder under features/xxx/components
-  - Sub components can live in that component folders, so small component do not need their own folder
-  - Place test next to their component file (same dir)
-  - Component sass should live in the same folder as component code
-- State logic & domain models should live in features/xxx/state
-- Containers (pages) can live in feature root features/xxx
-  - up for debate?
-
-## Props
+- Use spaces for for indentation
+
+### Naming
+- Use `PascalCase` for
+    - class names
```

Comment:
Not sure I follow here, is that for sass or emotion? I can't recall ever seeing class names in pascal case :thinking:

Reply:
TS class names :)

Reply:
Agree with Peter, a word or two explaining that we use PascalCase for TypeScript is needed. It is confusing to mix TypeScript naming rules with SASS naming rules - can they be separated out?